polaris-service/build.gradle.kts

```diff
@@ -108,6 +108,9 @@ dependencies {
   testImplementation(libs.assertj.core)
   testImplementation(libs.mockito.core)
   testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+
+  testRuntimeOnly(project(":polaris-eclipselink"))
```

Comment:
For now I think this could be okay, but I feel like we may not want these dependencies here forever. We have intentionally not included these dependencies in the service. A new project to include all these dependencies could make sense.

Reply:
If we create a separate project to include the dependencies, we still need to include such project here , right? This is for test only though.

Reply:
I will try to address in the following PRs to include the dependency if needed. I saw that we have `-PeclipseLink=$ECLIPSELINK`.